modules/nvidia_plugin/wheel/packages/openvino_nvidia/__init__.py

```diff
@@ -18,22 +18,44 @@ def _get_lib_file_extension() -> str:
         return "dylib"
 
 
-def _register_nvidia_plugin():
+def _register_nvidia_plugin(force=False):
     import openvino
     openvino_package_dir = os.path.dirname(os.path.abspath(openvino.__file__))
     openvino_package_libs_dir = os.path.join(openvino_package_dir, "libs")
     openvino_nvidia_gpu_package_dir = os.path.dirname(os.path.abspath(__file__))
-    openvino_nvidia_gpu_library = os.path.join(openvino_nvidia_gpu_package_dir, f"../libopenvino_nvidia_gpu_plugin.{_get_lib_file_extension()}")
+    openvino_nvidia_gpu_library = os.path.join(openvino_nvidia_gpu_package_dir, f"./lib/libopenvino_nvidia_gpu_plugin.{_get_lib_file_extension()}")
 
     xml_file = os.path.join(openvino_package_libs_dir, "plugins.xml")
+    xml_file_updated = False
     tree = ET.parse(xml_file).getroot()
     plugins = tree.find("plugins")
-    if all(plugin.get('name') != 'NVIDIA' for plugin in plugins.iter('plugin')):
-        plugins.append(Element('plugin', {'name': 'NVIDIA', 'location': openvino_nvidia_gpu_library}))
+    
+    if force:
+        for plugin in plugins:
+            if plugin.tag == "plugin" and plugin.get("name") == "NVIDIA":
+                plugins.remove(plugin)
+                plugins.append(Element('plugin', {'name': 'NVIDIA', 'location': openvino_nvidia_gpu_library}))
+                xml_file_updated = True
+    else:
+        if all(plugin.get('name') != 'NVIDIA' for plugin in plugins.iter('plugin')):
+            plugins.append(Element('plugin', {'name': 'NVIDIA', 'location': openvino_nvidia_gpu_library}))
+            xml_file_updated = True
+    
+    if xml_file_updated:
         with open(xml_file, "w") as f:
             f.write(ET.tostring(tree).decode('utf8'))
 
 
+def __getattr__(name):
+    if name == "install":
+        _register_nvidia_plugin()
+    elif name == "force_install":
+        _register_nvidia_plugin(True)
+    else:
+        raise AttributeError(f"module {__name__} has no attribute {name}")
+
+
 _register_nvidia_plugin()
 
-__version__ = "2024.1.0"
+
+__version__ = "2025.3.0"
```

Comment:
Perhaps we should be setting this dynamically based on `openvino` version, or at least set it to an upcoming `2025.4.0` if we plan to release it? I'm not sure how it should be done, @artanokhov do you know?

One way or another, if we'd like to set this dynamically, this is a topic for a different PR.

Reply:
> Perhaps we should be setting this dynamically based on `openvino` version, or at least set it to an upcoming `2025.4.0` if we plan to release it?

@p-wysocki Have not idea, I am not official maintainer of NVIDIA plugin, I am former developer of it and know pretty good its architecture.
In future I expect Intel will publish packages for `openvino-nvidia`, but for this proper CI should be configured